test/functional/apps/dashboard/group2/dashboard_filtering.ts

```diff
@@ -212,7 +212,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       });
 
       it('area, bar and heatmap charts', async () => {
-        await dashboardExpect.seriesElementCount(2);
```

Comment:
ℹ️ I am testing the x labels here instead of the y labels as there is this bug on std deviation https://github.com/elastic/kibana/issues/154523